processor/resourceprocessor/testdata/config.yaml

```diff
@@ -4,16 +4,14 @@ receivers:
 processors:
   # The following specifies an empty resource - it will have no effect on trace or metrics data.
   resource:
-  # The following specifies a non-trivial resource. Type "host" is used for Kubernetes node resources
-  # that expect the labels "cloud.zone", "k8s.cluster.name", "host.name" to be defined (although this
-  # is not enforced by the configuration logic).
+  # The following specifies a non-trivial resource. It sets resource attributes "cloud.zone", 
+  # "k8s.cluster.name" and "host.name" for every metric or trace being processed.
+  # If any of the attributes already have some values, they will be overridden.
   resource/2:
-    type: "host"
```

Comment:
Can you preserve the resource type for backward compatibility, please?
I believe it will be a label
```
"opencensus.resourcetype": "host"
```
based on https://github.com/open-telemetry/opentelemetry-collector/blob/474407c8377448577ba3cc29906aebe5373f7776/translator/conventions/opencensus.go#L24

Reply:
Yes, I was also thinking about it, but since we are in beta state and going through some breaking changes between releases anyway, decided to drop it right away. 
But probably you right, it might be too aggressive and we can mark it as deprecated for now ...

Reply:
done